content/00.front-matter.md

```diff
@@ -4,9 +4,14 @@
   Pandoc automatically inserts title from metadata.yaml, so it is not included in this template.
 ##}
 
-_A published version of this manuscript from 04 April 2018 is available at <https://doi.org/10.1098/rsif.2017.0387>.
-An effort is underway to update the manuscript.
-New authors and links to new sections are available in [GitHub Issue #959](https://github.com/greenelab/deep-review/issues/959)._
+<!-- include the Font Awesome library, per: https://fontawesome.com/start -->
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css">
+[
+[]{.fas .fa-info-circle .fa-lg} **Update Underway**<br>
```

Comment:
Now that you added a banner, should we remove ` [update in progress]` from the title?

Reply:
I'm not sure - google scholar seems to aggressively index this manuscript now, so I feel like we might want to leave that in the title.

Reply:
True.  Let's leave it in the title.